### ISSUE TITLE
Lifetime played.

### ISSUE THREAD

**Issue by darkniko** [genbtc/AutoTrimps#41]
Is there a way to know how much time i've played the game. Or maybe that AT can register that time? And tell that? I thought about adding up all portals but i only can see the last 16ish run times ..

**Comment by genbtc** [comment]
Nope. The data does not exist - and anything not in the graph is lost. Maybe post a suggestion on the trimps reddit to include this into the games own stats.

**Comment by darkniko** [comment]
Good thinking thnx!!!
